src/main/java/com/ing/data/cassandra/jdbc/CassandraDataSource.java

```diff
@@ -112,7 +118,8 @@ public CassandraDataSource(final List<ContactPoint> contactPoints, final String
      * @param localDataCenter   The local datacenter.
      */
     public CassandraDataSource(final List<ContactPoint> contactPoints, final String keyspace, final String user,
-                               final String password, final String consistency, final String localDataCenter) {
+                               final String password, final String consistency,
+                               final String localDataCenter, final String complianceMode) {
```

Comment:
I think it's better to not add an additional parameter to the constructor and just to have a setter to finally get something like this:
```java
DataSource ds = new CassandraDataSource(...);
ds.setComplianceMode("Liquibase");
Connection conn = ds.getConnection();
```
In the future, I'd like to simplify the `CassandraDataSource` constructors to keep the minimum of parameters and just have setters for any additional configuration.

Reply:
make sense

Reply:
done